src/project/FileTreeView.js

```diff
@@ -141,12 +142,12 @@ define(function (require, exports, module) {
          * this component, so we keep the model up to date by sending every update via an action.
          */
         handleInput: function (e) {
-            this.props.actions.setRenameValue(this.refs.name.getDOMNode().value.trim());
+            this.props.actions.setRenameValue(this.refs.name.value.trim());
 
             if (e.keyCode !== KeyEvent.DOM_VK_LEFT &&
                     e.keyCode !== KeyEvent.DOM_VK_RIGHT) {
                 // update the width of the input field
-                var domNode = this.refs.name.getDOMNode(),
+                var domNode = this.refs.name,
```

Comment:
This is old code... But do you think we can rename `domNode` to `node` to be consistent with the rest of the usage of `this.ref.XXX`?